openmdao/core/group.py

```diff
@@ -1302,6 +1302,15 @@ def _setup_connections(self):
                                         else:
                                             simple_warning(msg)
 
+                            for i in src_indices:
```

Comment:
Can you move your implementation over to the `auto_ivc` branch, and issue the pull request there?

Reply:
Moved